crates/parser/veryl.par

```diff
@@ -555,8 +555,13 @@ ClockDomain: BackQuote Identifier;
 // Statement
 // ----------------------------------------------------------------------------
 
-Statement: LetStatement
-         | IdentifierStatement
+StatementBlock: LBrace { StatementOrVarDeclaration } RBrace;
+
+StatementOrVarDeclaration: VarDeclaration | LetStatement | Statement;
+
+LetStatement: Let Identifier Colon [ ClockDomain ] ArrayType Equ Expression Semicolon;

```

Comment:
I think placing `LetStatement` at the previous place (line 569 in the original file) may be better because line 558-564 is related to `Statement*` category.